echo2.cc

```diff
@@ -5,6 +5,7 @@
 
 #include "common/common/assert.h"
 
+namespace Lyft {
```

Comment:
I don't really have a strong opinion here. Another option I guess is "EnvoyProxy". Very unlikely to conflict, but longer. Are there any other potential situations we could use an an example? @caniszczyk can you advise?